test/index-test.js

```diff
@@ -1,7 +1,133 @@
+const React = require('react');
+const { shallow } = require('enzyme');
+const sinon = require('sinon');
```

Comment:
When I run mocha, getting a can't find sinon module error...  (This happens on wip-solution too).